native-heapdump-examples/README.md

```diff
@@ -0,0 +1,129 @@
+# Creating a Heap Dump from a Native Executable
+
+You can create a heap dump of a running executable to monitor its execution. Just like any other Java heap dump, it can be opened with the [VisualVM](https://www.graalvm.org/latest/tools/visualvm/) tool. To enable heap dump support, native executables must be built with the --enable-monitoring=heapdump option.
+
+Then you can request heap dumps in the same way you can request them when your application runs on the JVM (for example, right-click on the process, then select "Heap Dump"). There are different ways to create heap dumps:
```

Comment:
This sentence should belong to the first paragraph:
"Then you can request heap dumps in the same way you can request them when your application runs on the JVM (for example, right-click on the process, then select "Heap Dump")." 

"There are different ways to create heap dumps:" is the next paragraph.